src/hotspot/share/gc/shenandoah/c2/shenandoahSupport.cpp

```diff
@@ -1734,11 +1734,26 @@ bool ShenandoahBarrierC2Support::identical_backtoback_ifs(Node* n, PhaseIdealLoo
   return true;
 }
 
+bool ShenandoahBarrierC2Support::merge_point_safe(Node* region) {
```

Comment:
This should probably be `is_merge_point_safe`, since it answers `bool`?

Reply:
There's an existing `merge_point_safe` method that validate merge points for split if (loopopts.cpp). That's why I named it like that.

Reply:
Ah, okay then!